packages/KTable/KTable.vue

```diff
@@ -9,7 +9,7 @@
             v-for="(column, index) in options.headers"
             :key="index"
             :class="!column.hideLabel && `${column.sortable ? 'sortable' : ''}${column.key === sortKey ? ' ' + sortOrder : ''}`"
-            @click="sortKey && $emit('sort', column.key)"
+            @click="column.sortable && $emit('sort', column.key, sortOrder)"
```

Comment:
🙌🏼

Reply:
You decided not to add the event to the header row?

Reply:
yea because it was non-trivial to pass in anything to the `<th>` beside the event itself